d2bs/kolbot/libs/config/Assassin.js

```diff
@@ -482,6 +482,7 @@ function LoadConfig() {
 	Config.UseBoS = false; // Set to true to use Burst of Speed prebuff. TODO: Casting in town + UseFade compatibility
 	Config.UseVenom = false; // Set to true to use Venom prebuff. Set to false if you don't have the skill and have Arachnid Mesh - it will cause connection drop otherwise.
 	Config.UseCloakofShadows = true; // Set to true to use Cloak of Shadows while fighting. Useful for blinding regular monsters/minions.
+	Config.AggressiveCloak = false; // Get into Cloak range on purpose, otherwise only casts if already close enough
```

Comment:
change comment to something like 
"Move into Cloak range or cast if already close"